ko/index.md

```diff
@@ -2,37 +2,28 @@
 layout: home
 
 hero:
-  name: 나인 크로니클
-  text: 개발자 포탈
-  tagline: 당신의 나인크로니클을 만드세요
-  actions:
-    - theme: brand
-      text: 모딩 시작하기
-      link: /ko/modding/getting-started
-    - theme: alt
-      text: 나인크로니클은 무엇인가?
-      link: /ko/general/what-is-nine-chronicles
-    - theme: alt
-      text: 네트워크 시작하기
-      link: /ko/network/local-network-tutorial/getting-started
-    - theme: alt
-      text: GitHub
-      link: https://github.com/planetarium
-    - theme: brand
-      text: 🎁 Modathon
-      link: /event/2024modathon
+  title: Nine Chronicles
+  titleTemplate: Build your Nine Chronicles
+
+  layout: home
+  aside: false
+  editLink: false
+  markdownStyles: false
 
-features:
-  - icon: 🔗
-    title: 완전 온체인
-    details: 나인 크로니클은 완전히 온체인으로 운영되어 게임 내 모든 거래와 상호작용의 투명성과 보안을 보장합니다.
-  - icon: 🌐
-    title: 탈중앙
-    details: 이 게임은 중앙 권한 없이 플레이어에 의해 관리되므로 진정한 탈중앙화 경험을 제공합니다.
-  - icon: 🌟
-    title: 커뮤니티 주도
-    details: 커뮤니티는 플레이어의 피드백과 기여를 통해 개발과 업데이트를 주도하며 게임의 미래를 만드는 데 중요한 역할을 합니다.
-  - icon: 🧩
-    title: 모딩 친화적
-    details: 나인 크로니클은 광범위한 모딩 도구와 지원을 제공하여 플레이어가 직접 콘텐츠와 수정 사항을 만들고 공유할 수 있습니다.
 ---
+
+<script setup>
+import Hero from '../.vitepress/theme/ko/Hero.vue'
+import WhatIsNineChronicles from '../.vitepress/theme/ko/WhatIsNineChronicles.vue'
+import ModdingFriendly from '../.vitepress/theme/ko/ModdingFriendly.vue'
+import OurVision from '../.vitepress/theme/ko/OurVision.vue'
+import JoinCommunity from '../.vitepress/theme/ko/JoinCommunity.vue'
+</script>
+
+<div class="VPHome">
+  <Hero/>
+  <WhatIsNineChronicles/>
+  <OurVision/>
+  <ModdingFriendly/>
+  <JoinCommunity/>
```

Comment:
Note, it is my personal opinion, not strong.

I think `ModdingFriendly` and `JoinCommunity` parts are more important to describe `Planetarium Labs`. So those parts should need to come further forward. (In English side, too)

```suggestion
  <ModdingFriendly/>
  <JoinCommunity/>
  <OurVision/>
```